packages/hardhat/test/fixture-projects/hardhat-project/typechain-types/common.ts

```diff
@@ -1,33 +1,69 @@
 /* Autogenerated file. Do not edit manually. */
 /* tslint:disable */
 /* eslint-disable */
-import type { Listener } from "@ethersproject/providers";
-import type { Event, EventFilter } from "ethers";
-
-export interface TypedEvent<
-  TArgsArray extends Array<any> = any,
-  TArgsObject = any
-> extends Event {
-  args: TArgsArray & TArgsObject;
-}
+/* Autogenerated file. Do not edit manually. */
```

Comment:
I think thit is a slight bug in ethers-v6 target that causes double preamble. Wanna have a look?

Reply:
Done.